sys/ztimer/Kconfig

```diff
@@ -23,6 +23,9 @@ config MODULE_ZTIMER_PERIPH_RTT
     depends on HAS_PERIPH_RTT
     select MODULE_PERIPH_RTT
 
+config MODULE_ZTIMER_NO_PERIPH_RTT
+    bool "Do not auto include RTT peripheral"
+
```

Comment:
I think these line could be somelines up (not hidden in a hidden by default submenu

#17099 has the equivalent option in line 10